### PR TITLE
Merge the initial project into master

### PR DESCRIPTION
**Pull request by ncthompson**

### CODE REVIEW

File: certgen/ca.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  },
+  "ca": {
+    "expiry": "262800h",
+    "pathlen": 1
+  },
+  "names": [
+    {
+      "C": "ZA",
+      "L": "Cape Town",
+      "O": "exampleorg.com",
```
Comment (jedrivisser) [inline certgen/ca.json]: Change to Die Bietse

File: cmd/mtls-server/main.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+import (
+	"crypto/x509"
+	"errors"
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"mtls-server/server"
```
Comment (jedrivisser) [inline cmd/mtls-server/main.go]: change modile name to fqdn

File: go.mod [hunk 3]
```diff
@@ -0,0 +1,5 @@
+module mtls-server
```
Comment (jedrivisser) [inline go.mod]: change to fqdn

File: go.mod [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+
+go 1.13
```
Comment (jedrivisser) [inline go.mod]: go 1.14

File: .github/workflows/build.yml [hunk 5]
```diff
@@ -0,0 +1,23 @@
+name: build
+on: [push, pull_request]
+jobs:
+  build:
+    name: verify
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.13
```
Comment (jedrivisser) [inline .github/workflows/build.yml]: go 1.14

File: certgen/ca.json (hunk 1 above)
Comment (ncthompson) [reply certgen/ca.json]: changed to diebietse.com

File: cmd/mtls-server/main.go (hunk 2 above)
Comment (ncthompson) [reply cmd/mtls-server/main.go]: now at github.com/diebietse/mtls-server

**Review by jedrivisser — APPROVED** [review]

**Pull request merged**